### PR TITLE
#3414 added -force option Remove-DbaOrphanUser

### LINKED ISSUE #3414: Repair-Orphan users does not contain Force parameter
### General Troubleshooting steps

- [X] Verified running the latest release of dbatools?
- [X] Verified errors are not related to permissions?
- [X] Can duplicate in new/clean PowerShell session (clean = `powershell -NoProfile`)?

### Version Information
 - Operating System (Name|Version): Windows 8.1
 - PowerShell Version: 5.0.10586.117
 - SQL Server (Edition|Version): All

## Steps to Reproduce
`Repair-DbaOrphanUser -SqlInstance <INSTANCE>-RemoveNotExisting`

Have one of the users own a schema. You then get this warning
```WARNING: [Remove-DbaOrphanUser][14:45:14] Schema 'SCHEMA' owned by user SOMEONE have 19 underlying objects. If you want to change the schemas' owner to 'dbo' and drop the user anyway, use -Force parameter. Skipping user 'SOMEONE'.                               ```

Reissue command with -Force parameter
`Repair-DbaOrphanUser -SqlInstance <INSTANCE>-RemoveNotExisting - Force`
New Error: 
```Repair-DbaOrphanUser : A parameter cannot be found that matches parameter name 'Force'.
At line:1 char:63
+ Repair-DbaOrphanUser -SqlInstance <INSTANCE>-RemoveNotExisting -Force
+                                                               ~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Repair-DbaOrphanUser], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Repair-DbaOrphanUser
```

### PR DESCRIPTION
**Pull request by garethnewman**
## Type of Change
Bug fix (non-breaking change, fixes #3414)

### Approach
Added the param $Force
Added an If to check for the param being used and ran the command with -Force, Else If Force wasn't used, run the original command.

### Commands to test
Repair-DbaOrphanUser -SqlInstance ComputerName -RemoveNotExisting -Verbose
Repair-DbaOrphanUser -SqlInstance ComputerName -RemoveNotExisting -Force -Verbose

### Screenshots

image

### CODE REVIEW

File: functions/Repair-DbaOrphanUser.ps1 [hunk 1]
```diff
@@ -206,12 +207,20 @@ function Repair-DbaOrphanUser {
 
                             #With the collection complete invoke remove.
                             if ($RemoveNotExisting -eq $true) {
-                                if ($Pscmdlet.ShouldProcess($db.Name, "Remove-DbaOrphanUser")) {
+                                if ($Force -eq $true) {
```
Comment (shaneis) [inline functions/Repair-DbaOrphanUser.ps1]: No need to do `$force -eq $true` here, you can just do `($force)` 😃

Comment (garethnewman) [reply functions/Repair-DbaOrphanUser.ps1]: Thanks @shaneis , is this the same too

if ($RemoveNotExisting -eq $true) ?

Comment (shaneis) [reply functions/Repair-DbaOrphanUser.ps1]: I think it works in more scenarios than just true / false but I just use it in those scenarios. Yeah, it works there as well 👍

**Comment by garethnewman** [comment]
The build error, is this how I would fix it for example?

        .PARAMETER Force
            If this switch is enabled, you will send the Force param to Remove-DbaOrphanUser, this will change ownership of objects to dbo and drop orphaned user.

**Comment by shaneis** [comment]
Yeah looks like it's expecting there to be comment based help (CBH) for the `$Force` parameter

I opened the details and this was at the bottom...

image

**Comment by garethnewman** [comment]
Yeah, that's what I saw, but wasn't sure if that was the right place for the comment that it is looking for :)
I'll fix it :+1:

**Comment by potatoqualitee** [comment]
Thank you for the PR @garethnewman ! and thank you for the review @shaneis 💯  Will merge once all tests pass

**Comment by garethnewman** [comment]
Those fails appear to be unrelated now?

**Comment by potatoqualitee** [comment]
yep! unrelated. working on it now 👍  thanks so much!

**Pull request merged**